sources/api/certdog/src/main.rs

```diff
@@ -0,0 +1,401 @@
+/*!
+  certdog is a tool to manage the trusted certificates store. It adds/removes
+  certificates from the final certificates bundle based on the configurations
+  in the API.
+*/
+
+#![deny(rust_2018_idioms)]
+
+#[macro_use]
+extern crate log;
+
+use simplelog::{Config as LogConfig, LevelFilter, SimpleLogger};
+use snafu::{OptionExt, ResultExt};
+use std::collections::HashMap;
+use std::env;
+use std::fmt::Write;
+use std::fs;
+use std::io::BufReader;
+use std::io::{BufRead, Seek};
+use std::path::Path;
+use std::process;
+use std::str::FromStr;
+use x509_parser;
+
+use model::modeled_types::Identifier;
+
+// FIXME Get from configuration in the future
+const DEFAULT_API_SOCKET: &str = "/run/api.sock";
```

Comment:
$1 to whoever who actually fixes this FIXME. 😢

Reply:
Or just removes the FIXMEs; I'm not sure the idea makes sense any more.

Reply:
A constants crate would at least consolidate the FIXME to one place, no?